### PR TITLE
Add message for migrated apps

### PR DESCRIPTION
**Pull request by tijmenb**
We are moving the tagging for some apps to content-tagger. When an app has been migrated (see settings.yml), tagging is disabled in panopticon.

This commit adds a message to the form for migrated apps with a link to the content-tagger, so that the transition will be seamless.

screen shot 2015-12-23 at 13 48 24

### CODE REVIEW

File: app/views/artefacts/_migrated_app.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="callout callout-success">
+  <div class="callout-title">
+    Improved tagging
+  </div>
+  <div class="callout-body">
+    Tagging for this item
+    <%= link_to "has moved", Plek.find('content-tagger') + "/content/#{artefact.content_id}" %>.
+  </div>
+</div>
+<br/>
```
Comment (fofr) [inline app/views/artefacts/_migrated_app.html.erb]: Why the extra line breaks? You can use the class `add-bottom-margin` to give the callout some bottom margin.

**Comment by fofr** [comment]
One minor comment, otherwise :+1:

**Comment by jackscotti** [comment]
I took this over @fofr. 

Removed the extra line breaks because they were superfluous:
- With:
screen shot 2016-01-04 at 10 14 38
- Without:
screen shot 2016-01-04 at 10 14 56

Also took the opportunity to add Pry.

**Comment by rboulton** [comment]
Looks good - thanks.

**Pull request merged**